.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Description
+
+What changed? Why? (the code + comments should speak for itself on the "how")
+
+Fun screenshots or a cool video or something are super helpful as well. If this touches platform-specific behavior, this is where test evidence should be collected.
+
+Any issues this pull request closes or pull requests this supersedes should be linked with `Closes #issuenumber`.
+
+## Meta
+
+Merge checklist:
+- [ ] Pull Request title is [short, imperative summary](https://cbea.ms/git-commit/) of proposed changes
+- [ ] The description documents the _what_ and _why_
+- [ ] If this PR touches photon-serde, all messages have been regenerated and hashes have not changed unexpectedly
+- [ ] If this PR touches configuration, this is backwards compatible with settings back to v2024.3.1
```

Comment:
Does the `v2024.3.1` need to be updated? In the original PR, we said that ["the template would be updated when we need to bump that"](https://github.com/PhotonVision/photonvision/pull/1561#discussion_r1845275774), and I don't know if we need to bump that since I wasn't paying much attention to what was happening in PV for a while.

Reply:
Nah, this is accurate for all of 2025